amboar/obmc-scripts/witherspoon-apply-power/README.md

```diff
@@ -0,0 +1,14 @@
+Description
+===========
+
+Avoids the chassis-on systemd target on Witherspoon. Instead, just whack the
```

Comment:
Is there some use case for this one?  Is obmcutil chassison not good enough?  Since obmcutil has chassiskill, should this function just be in there or are you thinking you'll make a remove-power script and put the function in there? A lot of times there's a reason for all the junk in chassis power on target (like voltage overrides for certain versions of chips).  How do we ensure fans get turned on with this?

Reply:
> Is there some use case for this one?

Yeah, though maybe it's better if I just keep this one in my local repo. I use it when I've netbooted a buildroot initrd and want to test the fan controller.

> Since obmcutil has chassiskill, should this function just be in there or are you thinking you'll make a remove-power script and put the function in there?

I wasn't feeling terribly fussed about symmetry.

> A lot of times there's a reason for all the junk in chassis power on target (like voltage overrides for certain versions of chips). 

That's probably a good reason for not merging it; so far I haven't cooked a system but it's certainly not going to help their life-span.

> How do we ensure fans get turned on with this?

In Witherspoon's case nothing special is required for the fans to start spinning, at least from the behaviour I've observed.

Again, its lack of completeness and general danger probably warrants keeping it out of the repo